multisig/proposals/vip-003/vip-003-sepolia.ts

```diff
@@ -0,0 +1,62 @@
+import { NETWORK_ADDRESSES } from "../../../src/networkAddresses";
+import { makeProposal } from "../../../src/utils";
+
+const { sepolia } = NETWORK_ADDRESSES;
+
+export const vip003 = () => {
+  return makeProposal([
+    {
+      target: sepolia.XVS_VAULT_PROXY,
+      signature: "_acceptAdmin()",
+      params: [],
+    },
+
+    {
+      target: sepolia.XVS_STORE,
+      signature: "acceptAdmin()",
+      params: [],
+    },
+
+    {
+      target: sepolia.ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [sepolia.XVS_VAULT_PROXY, "pause()", sepolia.NORMAL_TIMELOCK],
+    },
+
+    {
+      target: sepolia.ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [sepolia.XVS_VAULT_PROXY, "resume()", sepolia.NORMAL_TIMELOCK],
+    },
+
+    {
+      target: sepolia.ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [sepolia.XVS_VAULT_PROXY, "add(address,uint256,address,uint256,uint256)", sepolia.NORMAL_TIMELOCK],
+    },
+
+    {
+      target: sepolia.ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [sepolia.XVS_VAULT_PROXY, "set(address,uint256,uint256)", sepolia.NORMAL_TIMELOCK],
+    },
+
+    {
+      target: sepolia.ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [sepolia.XVS_VAULT_PROXY, "setRewardAmountPerBlock(address,uint256)", sepolia.NORMAL_TIMELOCK],
+    },
+
+    {
+      target: sepolia.ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [sepolia.XVS_VAULT_PROXY, "setWithdrawalLockingPeriod(address,uint256,uint256)", sepolia.NORMAL_TIMELOCK],
+    },
+
+    {
+      target: sepolia.XVS_VAULT_PROXY,
+      signature: "add(address,uint256,address,uint256,uint256)",
+      params: [sepolia.XVS, 100, sepolia.XVS, "61805555555555555", 604800],
```

Comment:
I would start with 0 rewards per block initially. We could set the real rate after sending XVS to the XVSStore contract, in a different VIP.

```suggestion
      params: [sepolia.XVS, 100, sepolia.XVS, "0", 604800],
```

Reply:
[a01da56](https://github.com/VenusProtocol/vips/pull/112/commits/a01da56458a4d2340f89f1478a031eaa6338c5dc)